mapbox/dr-ui
# Push text feedback to Sentry

This PR adds Sentry to the feedback module so we can manage/triage text feedback with Sentry. Since Segment captures operating system and browser, we can remove `detect-browser` and stop sending that information to Mode.

How to test
1. `npm install && npm start`
2. Navigate to: http://192.168.7.223:9966/Feedback
3. Submit text feedback using the first example
4. Check in Sentry https://sentry.io/organizations/mapboxcom/issues/?project=1848287 for the message.
5. Make sure the issue gets posted to #docs-feedback

Ref https://github.com/mapbox/documentation/issues/345
